src/pages/workspaces/workspace/notebooks/NotebookLauncher.js

```diff
@@ -70,35 +71,30 @@ class ReadOnlyMessage extends Component {
   }
 
   render() {
-    const { notebookName, workspace, workspace: { workspace: { namespace, name } } } = this.props
+    const { notebookName, workspace, workspace: { canCompute, workspace: { namespace, name } } } = this.props
```

Comment:
We use `workspace.canCompute` in line 60, is there a reason you de-structured `canCompute` specifically here?